### PR TITLE
chore(ci-lighthouse) - Add lighthouse support in CI

### PR DESCRIPTION
**Pull request by hybridx**
# Explain the feature/fix

-  Added lighthouse job in pipeline

## Does this PR introduce a breaking change

- No

## Screenshots

- N/A

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demoed and the design review approved?

### CODE REVIEW

File: .github/workflows/auto-deployments.yml [hunk 1]
```diff
@@ -106,3 +107,15 @@ jobs:
         run: |
           cd ${{ matrix.container }}
           npx @spaship/cli@latest deploy --apikey=${{ secrets.SPASHIP_API_KEY }} --env=${{ secrets.SPASHIP_DEPLOY_URL }}qa --ref=$(node -p "require('./package.json').version")
+
+      - name: Lighthouse CI
+        env:
+          NODE_TLS_REJECT_UNAUTHORIZED: 0
+          SSO_PASSWORD: ${{ secrets.SSO_PASSWORD }}
+          SSI_USERNAME: ${{ secrets.SSO_USERNAME }}
```
Comment (riginoommen) [inline .github/workflows/auto-deployments.yml:115]:
```suggestion
          SSO_USERNAME: ${{ secrets.SSO_USERNAME }}
```

**Review by riginoommen — CHANGES_REQUESTED** [review]

File: .github/workflows/manual-deployment.yml [hunk 2]
```diff
@@ -57,3 +57,23 @@ jobs:
...
+          NODE_TLS_REJECT_UNAUTHORIZED: 0
+          SSO_PASSWORD: ${{ secrets.SSO_PASSWORD }}
+          SSI_USERNAME: ${{ secrets.SSO_USERNAME }}
+          LH_HOST: ${{ secrets.LH_HOST }}
+          LHCI_BUILD_CONTEXT__CURRENT_BRANCH: ${{ steps.get-lighthouse-branch-name.outputs.LIGHTHOUSE_BRANCH_NAME }}
+
+        run: |
+          cd packages/$SPA_NAME
+          npm i puppeteer
+          npx @lhci/cli@0.8.2 autorun --upload.serverBaseUrl=${{ secrets.SERVER_BASE_URL }} --upload.token=${{ secrets.LIGHTHOUSE_TOKEN }} || echo 'Lighthouse assert failed'
```
Comment (riginoommen) [inline .github/workflows/manual-deployment.yml:79]:
```suggestion
          npx @lhci/cli@0.10.0 autorun --upload.serverBaseUrl=${{ secrets.SERVER_BASE_URL }} --upload.token=${{ secrets.LIGHTHOUSE_TOKEN }} || echo 'Lighthouse assert failed'
```

**Review by riginoommen — CHANGES_REQUESTED** [review]

File: .github/workflows/auto-deployments.yml (hunk 1 above)
Comment (hybridx) [reply .github/workflows/auto-deployments.yml]: Thank you, updated

**Review by deshmukhmayur — APPROVED** [review]
@hybridx Can you rebase and clean up the commits? Other than that, looks good to merge.

**Comment by hybridx** [comment]
> @hybridx Can you rebase and clean up the commits? Other than that, looks good to merge.

Updated the commit, thanks

**Pull request merged**